.golangci.yaml

```diff
@@ -16,8 +16,9 @@ linters-settings:
     # Default: ["end"]
     checks:
       - end
-      - record-error
-      - set-status
+      # disabling these because of false positives caused by early returns without errors
+      # - record-error
+      # - set-status
```

Comment:
don't disable it, if the linter given bad or unknown suggestion just put `//nolint` on the specific line